phpBB/language/en/acp/extensions.php

```diff
@@ -35,6 +35,22 @@
 // in a url you again do not need to specify an order e.g., 'Click %sHERE%s' is fine
 
 $lang = array_merge($lang, array(
+
+	'EXTENSION_ALREADY_INSTALLED'				=> 'The “%s” extension has already been installed.',
+	'EXTENSION_ALREADY_INSTALLED_MANUALLY'		=> 'The “%s” extension has already been installed manually.',
+	'EXTENSION_ALREADY_MANAGED'					=> 'The “%s” extension is already managed.',
```

Comment:
manage with composer an ext installed manually (basically: uninstall and reinstall with composer)